packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -54,4 +60,23 @@ export class MeterProvider implements api.MeterProvider {
 
     return this._meters.get(key)!;
   }
+
+  shutdown(cb: () => void = () => {}): void {
+    this._shutdownAllMeters().then(() => {
+      setTimeout(cb, 0);
+    });
+    if (this._cleanNotifyOnGlobalShutdown) {
+      this._cleanNotifyOnGlobalShutdown();
+      this._cleanNotifyOnGlobalShutdown = undefined;
+    }
+  }
+
+  private async _shutdownAllMeters() {
+    if (this._config.exporter) {
+      this._config.exporter.shutdown();
+    }
+    await Promise.all(
```

Comment:
why keeping the whole function an async if the only usage of "await" is done for "Promise.all" just return "Promise.all" and no need to make it async as it will have exactly the same flow, but when exporting to es5 it will not have "generators code"